### ISSUE TITLE
Contributing Guidelines

### ISSUE THREAD

**Issue by CommonClimate** [LinkedEarth/FreqFun#13]
We need contributing guidelines so we can open up contributions to a wider community.

Here are some desiderata gathered so far:

- notebooks need a descriptive title and a name that (as best as possible) evokes that title
- notebooks need to be signed by the contributor, with a URL that directs to a professional profile of their choice (e.g. website, github profile, Google Scholar, etc)
- notebooks need an introduction that lays out the scientific context and the technical skills being illustrated 
- new notebooks can be developed in dedicated branches or other repos, and introduced to PaleoBooks by a pull request 
- we need to decide on a process for reviewing notebooks, which probably involves testing them a) in myBinder b) on the hub.  Only once these two conditions are met should notebooks be merged into master

**Comment by jordanplanders** [comment]
@CommonClimate What's your read on this issue? I think this is broadly covered by the Paleobook contribution guidelines. Given we are allowing people to maintain more control over their books, we've let go of guaranteeing that they work on myBinder, but I think the LLM pals now provide some of this support.

**Comment by CommonClimate** [comment]
Again, this was indeed done a long time ago. You should get in the habit of closing issues as you do things! 

The one remaining issue on this front is to come up with a more sensible labeling system, and notebook/repo titles that more plainly advertise the contents than "LMR-CMIP6 Paleobook" (I believe "Data-model comparison over the Common Era" was the title I originally suggested , and I maintain that it would draw a bigger audience than an acronym soup)
